src/Runner.Listener/BrokerMessageListener.cs

```diff
@@ -74,6 +74,7 @@ public async Task<TaskAgentMessage> GetNextMessageAsync(CancellationToken token)
                 try
                 {
                     message = await _brokerServer.GetRunnerMessageAsync(_getMessagesTokenSource.Token,
+                                                                        null,
```

Comment:
We should add the sessionID here, right?

Reply:
yes. Though I'm kind of thinking BrokerMessageListener might eventually be deprecated.